### PR TITLE
support github ci and e2e test

### PR DESCRIPTION
**Pull request by huiwq1990**
/kind enhancement
> /kind good-first-issue
> /kind feature
> /kind question
> /kind design
> /sig ai
> /sig iot
> /sig network
> /sig storage
> /sig storage

#### What this PR does / why we need it:

Add e2e test and github ci

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@huiwq1990 please fix the conflicting files in ci pipeline

File: hack/lib/release-images.sh [hunk 1]
```diff
@@ -100,6 +102,11 @@ push_images() {
     docker push ${yurt_component_image}
 }
 
+kindload_images() {
+    build_images
```
Comment (rambohe-ch) [inline hack/lib/release-images.sh:106]: we have used `docker buildx` to replace `build_images` for building images, so you should to adapt the changes in  this pull request.

Comment (rambohe-ch) [reply hack/lib/release-images.sh:106]: so you can add a step in `e2e` workflow for building images by `make docker-build`

File: hack/lib/release-images.sh [hunk 2]
```diff
@@ -100,6 +102,11 @@ push_images() {
...
+    kind load docker-image ${yurt_component_image} || { echo >&2 "kind not installed or error loading image: $(yurt_component_image)"; exit 1; }
```
Comment (rambohe-ch) [inline hack/lib/release-images.sh:107]: which node should we load image onto? maybe we need to specify the node by `--nodes xxx`

**Comment by huiwq1990** [comment]
close, new pr: https://github.com/openyurtio/yurt-app-manager/pull/107

**Pull request closed without merging**